modular_skyrat/modules/customization/modules/clothing/~donator/donator_clothing.dm

```diff
@@ -1373,3 +1373,65 @@
+	desc = "Hey, free college!"
+	icon_state = "officerhat"
+	icon = 'modular_skyrat/master_files/icons/donator/obj/clothing/hats.dmi'
+	worn_icon = 'modular_skyrat/master_files/icons/donator/mob/clothing/head.dmi'
+	worn_icon_state = "officerhat"
+
+// Donation reward for M97screwsyourparents
+/obj/item/clothing/suit/toggle/recruiter_jacket
+	name = "recruiter jacket"
+	desc = "Hey, free college!"
+	icon = 'modular_skyrat/master_files/icons/donator/obj/clothing/suits.dmi'
+	worn_icon = 'modular_skyrat/master_files/icons/donator/mob/clothing/suit.dmi'
+	icon_state = "officerjacket"
+	base_icon_state = "officerjacket"
+
+// Donation reward for M97screwsyourparents
+/obj/item/clothing/under/recruiter_uniform
+	name = "recruiter uniform"
+	desc = "Hey, free college!"
+	icon = 'modular_skyrat/master_files/icons/donator/obj/clothing/uniform.dmi'
+	icon_state = "oldmarine_whites"
+	worn_icon = 'modular_skyrat/master_files/icons/donator/mob/clothing/uniform.dmi'
+	worn_icon_digi = null
+
+//Donation reward for Tetrako
+/obj/item/clothing/under/nt_idol_skirt
+	name = "\improper NT Idol's suitskirt"
+	desc = "This outfit resembles much the same as other ranking NT Officers, but comes with certain bells and whistles, like frills around the dress, slight puffs around the shoulders and most importantly, several golden buckles to accentuate the green! The only thing fit for NT's very own idols to wear!"
+	icon = 'icons/obj/clothing/under/centcom.dmi'
+	worn_icon = 'icons/mob/clothing/under/centcom.dmi'
+	icon_state = "centcom_skirt"
+	inhand_icon_state = "dg_suit"
+	female_sprite_flags = FEMALE_UNIFORM_TOP_ONLY
+	body_parts_covered = CHEST|GROIN|ARMS
+	supports_variations_flags = CLOTHING_DIGITIGRADE_VARIATION_NO_NEW_ICON
+
+// Donation reward for SlippyJoe
+/obj/item/clothing/accessory/hypno_watch
+	name = "cracked pocket watch"
+	desc = "A shining pocket watch, cast in gold and embossed with metallic swirls that almost seem  amethyst under the right light... There's a button on the top to unlatch the front panel, although all that's inside is a layer of cracked glass, the argent hands stuck pointing to 7:07 PM. The brushed silver of these arrows almost seems to swirl if one's gaze lingers for too long. Despite its inert apparence, the eerie mechanical sound of gears turning and clicking in place seems to quietly ring out from the artifact. In the right hands, [span_boldwarning("Who knows what it could be used for?")]"
+	icon = 'modular_skyrat/master_files/icons/donator/obj/custom.dmi'
+	worn_icon = 'modular_skyrat/master_files/icons/donator/mob/clothing/custom_w.dmi'
+	lefthand_file = 'modular_skyrat/master_files/icons/donator/mob/inhands/donator_left.dmi'
+	righthand_file = 'modular_skyrat/master_files/icons/donator/mob/inhands/donator_right.dmi'
+	worn_icon_state = "pocketwatch"
+	icon_state = "pocketwatch"
+	inhand_icon_state = "pocketwatch"
+
+	var/list/spans = list("velvet")
+	actions_types = list(/datum/action/item_action/hypno_whisper)
+
```

Comment:
There might be a parameter or two to `examine()` but I think you get the jist of it.
```suggestion
/obj/item/clothing/accessory/hypno_watch
	name = "cracked pocket watch"
	desc = "A shining pocket watch, cast in gold and embossed with metallic swirls that almost seem  amethyst under the right light... There's a button on the top to unlatch the front panel, although all that's inside is a layer of cracked glass, the argent hands stuck pointing to 7:07 PM. The brushed silver of these arrows almost seems to swirl if one's gaze lingers for too long. Despite its inert apparence, the eerie mechanical sound of gears turning and clicking in place seems to quietly ring out from the artifact. In the right hands, "
	icon = 'modular_skyrat/master_files/icons/donator/obj/custom.dmi'
	worn_icon = 'modular_skyrat/master_files/icons/donator/mob/clothing/custom_w.dmi'
	lefthand_file = 'modular_skyrat/master_files/icons/donator/mob/inhands/donator_left.dmi'
	righthand_file = 'modular_skyrat/master_files/icons/donator/mob/inhands/donator_right.dmi'
	worn_icon_state = "pocketwatch"
	icon_state = "pocketwatch"
	inhand_icon_state = "pocketwatch"

	var/list/spans = list("velvet")
	actions_types = list(/datum/action/item_action/hypno_whisper)

/obj/item/clothing/accessory/examine()
	+= span_boldwarning("who knows what it could be used for?")

```